For some reason `poetry.python.pkgs.deepdiff` fails to build:
```
=================================== FAILURES ===================================
______________________ TestDiffOther.test_repeated_timer _______________________

self = <tests.test_diff_other.TestDiffOther object at 0x1133c97e0>
MockDiffLevel = <MagicMock name='DiffLevel' id='4625369840'>

    @mock.patch('deepdiff.diff.DiffLevel', side_effect=SlowDiffLevel)
    def test_repeated_timer(self, MockDiffLevel):
        t1 = [1, [2]]
        t2 = [1, [3]]

        progress_logger = mock.Mock()
        DeepDiff(t1, t2, log_frequency_in_sec=0.02, progress_logger=progress_logger)
>       assert PROGRESS_MSG.format(0, 0, 0) == progress_logger.call_args[0][0]
E       TypeError: 'NoneType' object is not subscriptable

tests/test_diff_other.py:28: TypeError
```

--- a/pkgs/tools/package-management/poetry/default.nix
+++ b/pkgs/tools/package-management/poetry/default.nix
@@ -10,6 +10,9 @@ let
       poetry = self.callPackage ./unwrapped.nix { };
 
       # version overrides required by poetry and its plugins
+      deepdiff = super.deepdiff.overridePythonAttrs (old: rec {
+        doCheck = false;
+      });
       poetry-core = super.poetry-core.overridePythonAttrs (old: rec {
         version = "1.7.0";
         src = fetchFromGitHub {
@@ -18,6 +21,7 @@ let
           rev = version;
           hash = "sha256-OfY2zc+5CgOrgbiPVnvMdT4h1S7Aek8S7iThl6azmsk=";
         };
+        patches = [ ];
       });
     } // (plugins self);
   };

